### PR TITLE
Update docs for adding camera/displays to CMIO

### PR DESCRIPTION
**Pull request by 6by9**
Followup to #1755 to update the docs.

### CODE REVIEW

File: hardware/computemodule/cmio-display.md [hunk 1]
```diff
@@ -47,26 +63,25 @@ Please note that the wiring is slightly different from that on the Camera page,
...
 
 1. Reboot for the `dt-blob.bin` file to be read.
 
 	![Camera Preview on the 7 inch display](images/CMIO-Cam-Disp-Example.jpg)
-	(Please note this image needs to be updated to show two Camera Modules, or have the extra jumper leads removed)
+	(Please note this image needs to be updated to show two Camera Modules and the standard wiring)
 
 ### Software support
 
 There is no additional configuration required to enable the touchscreen. The touch interface should out work of the box once the screen is successfully detected.
```
Comment (JamesH65) [inline hardware/computemodule/cmio-display.md:79]: I know its not in the change, but there is an extra 'out' in here

Comment (6by9) [reply hardware/computemodule/cmio-display.md]: Fixed. The "out" is needed, but is misplaced by a word (off-by-one error!)

**Comment by 6by9** [comment]
Thanks James.
Who needs to be pinged to get CM4IO photos done?

If we can get a production CM4IO board from Dominic then I'm happy to go into the office to work with whoever to get photos done. I don't have a suitable CM4IO board or I'd tidy up and do them here.

**Comment by JamesH65** [comment]
Hmm, I guess you need to go via comms so @helenlynn

**Comment by helenlynn** [comment]
Yes that's our team. Have a word with Brian @6by9, he or Fiacre can sort you out.

**Pull request merged**